README.md

```diff
@@ -22,7 +22,7 @@ You can leverage this plugin for your project in one of two ways:
 ## Applying the published plugin
 
 The Common Custom User Data Gradle plugin is available in the [Gradle Plugin Portal](https://plugins.gradle.org/plugin/com.gradle.common-custom-user-data-gradle-plugin). This
-plugin requires the [Develocity Gradle plugin](https://plugins.gradle.org/plugin/com.gradle.enterprise) to also be applied in your build in order to have an effect.
+plugin requires the [Develocity Gradle plugin](https://plugins.gradle.org/plugin/com.gradle.develocity) to also be applied in your build in order to have an effect.
```

Comment:
Let's not make this change quite yet since this isn't published

Reply:
That was the idea, but now I think about merging PRs before that. With all the CI plugin updates, I just have too many moving parts on my plate, and would like to close them one-by-one.

So, I reverted this change for now in https://github.com/gradle/common-custom-user-data-gradle-plugin/pull/273/commits/ca43d566bd4ca62dbf66a167b1b3e0b41df657dd